tools/validators/instance_validator/tests/fake_instances/GOOD/update_change_subset_of_entities.yaml

```diff
@@ -31,7 +31,7 @@ SDC_EXT-18-GUID:
 SDC_EXT-17-GUID:
   type: HVAC/SDC_EXT
   code: SDC_EXT-17
-  cloud_device_id: "foobar"
+  cloud_device_id: "2619178366980754"
```

Comment:
change cloud_device_id to something more generic like 1234567890123456

Reply:
Done.